docs/authors.rst

```diff
@@ -0,0 +1,22 @@
+.. raw :: html
+
+    <!-- Generated by generate_authors_table.py -->
+    <div class="sk-authors-container">
+    <style>
+      img.avatar {border-radius: 5px;}
+    </style>
+    <div>
+    <a href='https://github.com/solegalli'><img src='https://avatars.githubusercontent.com/solegalli?v=4' class='avatar' width="120"
+     height="120" /></a> <br />
+    <p>Soledad Galli</p>
+    </div>
+    <div>
+    <a href='https://github.com/christophergs'><img src='https://avatars.githubusercontent.com/christophergs?v=4' width="120"
+     height="120"class='avatar' /></a> <br />
+    <p>Chris Samiullah</p>
+    </div>
+    <div>
+    <a href='https://github.com/nicogalli'><img src='https://avatars.githubusercontent.com/nicogalli?v=4' class='avatar'width="120"
```

Comment:
@NicoGalli I added you as a core contributor, would you mind uploading a photo or cartoon to github? it will display automatically. Thank you!